playbooks/tasks/portal-firewall-ufw-setup.yml

```diff
@@ -41,17 +41,22 @@
         - 443
         - 26257
         - 27017
-    
+
+    - name: Allow outgoing traffic to docker local network
+      community.general.ufw:
+        rule: "allow"
+        proto: "any"
+        direction: "out"
+        to_ip: "{{ item }}"
+      loop: "{{ local_networks_allowed }}"      
+
```

Comment:
I think this is ok. Because above is where we do the general allow and deny rules which would impact each other. But now with the specific ports it should be fine to do it this way.

Reply:
> As written in discord, for `non-default` rules, first rule match wins, so `local_networks_allowed` (narrow `10.10.10.0/24`) must be allowed before we deny `local_networks_denied` incl. wide `10.0.0.0/8`.

I thought it was other way around so please make sure it's properly tested on dev server before we deploy to prod

Reply:
I thought so as well but here's a quote from the docs:
`This puts the specific rules first and the generic second. Once a rule is matched the others will not be evaluated (see manual below) so you must put the specific rules first.` (https://help.ubuntu.com/community/UFW#Advanced_Example)

We should definitely test this manually on the dev cluster.